crates/protocol/src/package/mod.rs

```diff
@@ -8,7 +8,7 @@ mod model;
 mod validate;
 
 pub use model::{PackageEntry, PackageRecord, Permission};
-pub use validate::{Head, PackageState, Release, ReleaseState, ValidationError};
```

Comment:
There's still an inconsistency here between `PackageState`/`PackageValidationError` and `Release`/`ReleaseState`, which I think is a good focal point for the main question we have to answer.

Which of the following do we want in the codebase?
1. all names get log-name prefixing including `PackageRelease` & `PackageReleaseState`;
2. some names get log-name prefixing, but others like `Release` and `ReleaseState` don't;
3. no names are log-name prefixed and we consistently use `package::` and `operator::` for disambiguation like we do today.

I'm not a fan of the inconsistency of 2, without trying them both I don't have a huge preference between 1 & 3. What do other people think?

@lann @peterhuene

Reply:
I'm fine with 1 or 3. I would probably pick 1 for this particular case.

Reply:
Option 1 pushed.